modules/lang-painless/src/main/java/org/opensearch/painless/DefBootstrap.java

```diff
@@ -204,7 +215,10 @@ protected MethodHandle computeValue(Class<?> receiverType) {
                     try {
                         return lookup(flavor, name, receiverType).asType(type);
                     } catch (Throwable t) {
-                        Def.rethrow(t);
+                        switch (t) {
+                            case Exception e -> throw new WrappedCheckedException(e);
```

Comment:
Not any better, but may be we could use `UndeclaredThrowableException` instead of `WrappedCheckedException`, just a suggestion, thanks @andrross